var/spack/repos/builtin/packages/hybrid-lambda/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class HybridLambda(AutotoolsPackage):
+    """Hybrid-Lambda is a software package that can simulate gene trees
+       within a rooted species network or a rooted species tree under the
+       coalescent process. 
+       
+       The main feature of this program is that users can
+       choose to use the standard Kingman coalescent process, which produces
+       bifurcating genealogies, or two other Lambda coalescent processes,
+       which produce multifurcating genealogies. The other feature is that
+       hybrid sim uses extended Newick formatted strings to make it easier to
+       represent hybridization events between species."""
+
+    homepage = "https://github.com/hybridLambda/hybrid-Lambda"
+    git      = "https://github.com/hybridLambda/hybrid-Lambda.git"
+
+    version('dev', submodules=True)
```

Comment:
```suggestion
    version('develop', submodules=True)
```

Reply:
looks good, dev was from their build docs, but develop works as well

Reply:
Just to be clear, `version('master', submodules=True)` will download the same version as the previous two. If you don't specify a `branch`, it will download the default branch. The first arg is just the version name that Spack calls it.